tests/Application/config/packages/doctrine_migrations.yaml

```diff
@@ -0,0 +1,5 @@
+doctrine_migrations:
+    dir_name: "%kernel.project_dir%/app/migrations"
+    namespace: Sylius\Migrations
```

Comment:
We should keep in mind to switch to `DoctrineMigrations` namespace (it can be for sure done in next PR, as it requires some changes outside the test application)

Reply:
Oh, in fact, it was already changed 😄 https://github.com/Sylius/InvoicingPlugin/pull/62
So the config is invalid